lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -23,6 +24,7 @@ export type AccountInfo = {
     username: string;
     localAccountId: string;
     name?: string;
+    idToken?: string;
```

Comment:
Why is this param optional in `AccountInfo` and mandated in `AccountEntity`?

Reply:
@sameerag I made a mistake - I believe it should be mandated in both. Do you agree?